### ISSUE TITLE
Add renumberMesh command in the solution component

### ISSUE THREAD

**Issue by TheodoreGalanos** [ladybug-analysis-tools/butterfly#150]
This is a minor thing but it might provide increase in speed, which from my experience so far it's been disappointing in Windows.

The command is truly simple, only thing  typed is renumberMesh -overwrite and we can add it automatically to the run component right before running the case. It can also use a dictionary input which adds more options (although defaults are more than fine): https://develop.openfoam.com/Development/OpenFOAM-plus/blob/master/applications/utilities/mesh/manipulation/renumberMesh/renumberMeshDict

It can provide a good increase in speed for linear solvers I believe.

Kind regards,
Theodore.

**Comment by mostaphaRoudsari** [comment]
@TheodoreGalanos we can do this and it is fairly simple to implement but what about adding a component that let's you run any OpenFOAM command? Something like this:

![image](https://cloud.githubusercontent.com/assets/2915573/21727343/059c70cc-d40f-11e6-80c3-5fef1a7a1767.png)

**Comment by TheodoreGalanos** [comment]
Yes @mostaphaRoudsari that can work. Only thing is that we make BF aware of when to run each command. Maybe a simple flag 'before' and 'after' is enough to tell it to run it before the run or after. For example, renumberMesh should be run right before. Although I guess maybe that's not necessary if this component actually runs it on the spot as I understand it does. 

Btw this actually looks a lot like what I had in mind as an example of a custom function object component in #149, or at least a custom implementation of it.

**Comment by TheodoreGalanos** [comment]
@mostaphaRoudsari what is the status of this? 

It could help considerably with simulation speed if we implement it. Just need to test it quickly to make sure BF can run the case after. Also, the overwrite flag is useful to avoid creating a new time but will also change the 0 folder (adding information and turning them to zip files). Should be ok though since BF can re-write that folder anytime.
